Fix Tab type in TabPane

<!--
Thank you for contributing to Dojo 2.

Our issue tracker is for bugs for Dojo 2.

Please make sure you have read our Contributing Guidelines
available at: https://github.com/dojo/meta/blob/master/CONTRIBUTING.md

For general questions and discussion, join us on Gitter.im at: https://gitter.im/dojo/dojo2
-->

**Bug** <!-- delete as appropriate -->

Grunt dist not currently working due to wrong `Tab` type

FYI it wasn't because it was the "wrong" `Tab` it was because the type that was created `type Tab = WidgetBase<TabProperties>` was not exported.

Causing: `src/tabpane/TabPane.ts(43,75): error TS4020: 'extends' clause of exported class 'TabPane' has or is using private name 'Tab'.`

--- a/src/tabpane/TabPane.ts
+++ b/src/tabpane/TabPane.ts
@@ -5,6 +5,7 @@ import { ThemeableMixin, ThemeableProperties, theme } from '@dojo/widget-core/mi
 import { v, w } from '@dojo/widget-core/d';
 import { WidgetBase } from '@dojo/widget-core/WidgetBase';
 import TabButton from './TabButton';
+import Tab from './Tab';
 import uuid from '@dojo/core/uuid';
 
 import * as css from './styles/tabPane.m.css';
@@ -37,7 +38,6 @@ export interface TabPaneProperties extends ThemeableProperties {
 };
 
 export const TabPaneBase = ThemeableMixin(WidgetBase);
-type Tab = WidgetBase<TabProperties>;
 
 @theme(css)
 export default class TabPane extends TabPaneBase<TabPaneProperties, WNode<Tab>> {

--- a/src/tabpane/TabPane.ts
+++ b/src/tabpane/TabPane.ts
@@ -1,11 +1,10 @@
 import { assign } from '@dojo/core/lang';
 import { DNode, WNode } from '@dojo/widget-core/interfaces';
-import { TabProperties } from './Tab';
+import Tab, { TabProperties } from './Tab';
 import { ThemeableMixin, ThemeableProperties, theme } from '@dojo/widget-core/mixins/Themeable';
 import { v, w } from '@dojo/widget-core/d';
 import { WidgetBase } from '@dojo/widget-core/WidgetBase';
 import TabButton from './TabButton';
-import Tab from './Tab';
 import uuid from '@dojo/core/uuid';
 
 import * as css from './styles/tabPane.m.css';

